index.html

```diff
@@ -1,11 +1,88 @@
-<!DOCTYPE html>
+<!doctype html>
 <html lang="en">
-  <head>
+<head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Homework 1</title>
-  </head>
-  <body>
-    <h1>Start developing here</h1>
-  </body>
+</head>
+<body>
+<header>
+    <h3><a href="https://web-studio"><span>Web</span>Studio</a></h3>
+    <nav>
+        <ul>
+            <li><a href="https://studio">Студія</a></li>
+            <li><a href="https://portfolio">Портфоліо</a></li>
+            <li><a href="https://contacts">Контакти</a></li>
+        </ul>
+    </nav>
+    <div>
+        <span>info@devstudio.com</span>
+        <span>+38 096 111 11 11</span>
```

Comment:
these are also links

```suggestion
        <a href="mailto:info@devstudio.com">info@devstudio.com</a>
        <a href="tel:380961111111">+38 096 111 11 11</a>
```